src/app/public/modules/toggle/toggle.component.scss

```diff
@@ -0,0 +1,53 @@
+/* Hide default checkbox */
+.sky-switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.sky-toggle-slider {
+  position: relative;
+  width: 47px;
+  height: 24px;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border: 1px solid #cccccc;
+  border-radius: 13px;
+  background-color: #e5e6e7;
```

Comment:
Seems like most of these codes aren't in `@skyux/theme` but there are similar colors so I guess they might have changed since the design was made. For example, the design says the blue color for the toggle is `$bb-color-blue` which should be `#1c84c6` but that code isn't in `@skyux/theme` and it seems the naming is outdated. Should I switch the colors out with similar ones that are now in `@skyux/theme`?

Reply:
gray fill - $sky-background-color-neutral-light
gray border - $sky-border-color-neutral-medium

blue fill - $sky-background-color-info-light
blue border - $sky-highlight-color-info

disabled fill - $sky-background-color-disabled
disabled fill for thumb - a decent question, actually
   -could be $sky-background-color-neutral-light
   -could be $sky-border-color-neutral-light for slightly more contrast against active state